### PR TITLE
Add Taiwanese brands 添加多個臺灣品牌

### PR DESCRIPTION
**Pull request by iigmir**
### Taiwanese Enterprise Groups

1. Mercuries Group:
    1. Add the `operator` tag to Mercuries Food (Taiwan), Mercuries Food (Japan), Tonkatsu, and Simple Mart, to link them into the Mercuries Group.
    2. Add Napoli Pizza and Napoli Fried Chicken. Napoli Pizza sells pizza and fried chicken, but Napoli Fried Chicken only sells fried chicken. 
2. Feastogether Group: Add Eatogether, Fruitfulfood, Xujisunrise, Xiao Free, Kaifun, Siam More, and Jin Zhu. 
3. TTFB Restaurant Group: Add Thai Town, Very Thai, Bheart Noodles, Shann Rice Bar, and 1010 Restaurant.
4. Sushi Express: Add Sushi Take Out, Teishoku 8, Magic Touch, and Sushi Plus.
5. Uni-President: Add Mech Smile and Santa Cruz.

### 臺灣的企業集團相關（Taiwanese Enterprise Groups 的中文翻譯）

1. 三商企業集團：
    1. 給三商巧福 (臺灣)、三商巧福 (日本)、福勝亭、美廉社添加 `operator` tag，把他們與三商企業集團的關係連繫起來。
    2. 添加拿坡里披薩與拿坡里炸雞。拿坡里披薩有賣披薩與炸雞、但拿坡里炸雞只賣炸雞。
2. 饗賓餐旅集團：添加饗食天堂、果然匯、旭集、小福利、開飯、饗泰多、真珠。
3. 瓦城泰統集團： 添加瓦城、非常泰、大心、時時香、1010湘。
4. 爭鮮： 添加爭鮮gogo、定食8、Magic Touch、爭鮮Plus。
5. 統一企業集團：添加速邁樂加油站與聖德科斯。

### Taiwanese Hotel bands (臺灣的飯店品牌)

Add the following hotel bands (添加以下飯店品牌):

1. 長榮國際連鎖酒店 (Evergreen International Hotels)
2. 凱撒飯店 (Caesar Park)
3. 老爺酒店 (Hotel Royal)
4. 福容大飯店 (Fullon Hotels)
5. 神旺大飯店 (San Want Hotels)
6. 國賓大飯店 (Ambassador Hotel)
7. 康橋商旅 (Kindness Hotel)

### Other(其他)

Add the following bands (添加以下品牌):

1. 山隆加油站
2. 金石堂書店
3. 諾貝爾圖書城
4. 墊腳石圖書文化廣場
5. 光南大批發
6. iColor

---

臺灣的企業集團真的很多，縮排好像因此壞了……

### CODE REVIEW

**Comment by bhousel** [comment]
> [Delete brands under 10](https://github.com/osmlab/name-suggestion-index/pull/9875/commits/296e718ba38b61c98d74a268a670b8720a25fba8)

I don't think you necessarily need to delete the brands < 10.  If they are notable and have a wikidata, it's ok.

**Comment by iigmir** [comment]
> If they are notable and have a wikidata, it's ok.

Thanks, if it is ok, I think some of the entries can be retrieved.

File: data/brands/amenity/restaurant.json [hunk 1]
```diff
@@ -3756,6 +4793,7 @@
       "locationSet": {
         "include": ["br", "ca-qc.geojson", "fr"]
       },
+      },
```
Comment (LaoshuBaby) [inline data/brands/amenity/restaurant.json:4796]:
```suggestion
```

**Comment by matkoniecz** [comment]
Q130214352 for Bheart Noodles got deleted on Wikidata. Can you arrange undeletion? See also https://github.com/osmlab/name-suggestion-index/pull/11106

**Comment by Snowysauce** [comment]
In addition to Bheart Noodles, the Wikidata items for at least three other brands added in this PR were also deleted. See #10303 for more information

**Pull request merged**